modules/json/json-parser.c

```diff
@@ -144,25 +176,7 @@ json_parser_process_single(struct json_object *jso,
         }
       break;
     }
-    case json_type_null:
-
-      /* null values are represented as an empty string (so when expands to
-       * "nothing" in type unaware situations, while type-aware consumers
-       * may reproduce it as a NULL).
-       *
-       * I was thinking about using a very specific string representation
-       * (e.g.  "NULL" as a value), however if we encounter a null at a
-       * place where we might also have a sub-object, and we explicitly
-       * reference a field within that sub-object, we would get an empty
-       * string in that case too (due to the key being unknown).  All in
-       * all, the base case is I think to use an empty string as value, and
-       * LM_VT_NULL as type.  Type aware consumers would ignore the string
-       * anyway.
-       */
-
       parsed = TRUE;
```

Comment:
In e23ddc690751da33f13fa19edf54a7436865b10a, the extraction seems to be incomplete as this line remained here.

Reply:
thanks, fixed this up.